base/deprecated.jl

```diff
@@ -539,6 +539,63 @@ end
 
 export MathConst, @math_const
 
+# 11551
+function utf16_is_surrogate(chr::UInt16)
```

Comment:
It might not be worth the trouble. Since JSON has been fixed, the need for this PR is questionable.